### PR TITLE
Code refactored and directory structure updated

### PR DESCRIPTION
**Pull request by ankushchavan1**
- The docker-compose.yml file removed
- Templates removed
- news app removed
- README updated
- Directory structure updated

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -114,7 +114,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-learndjango/.env
+server/config/settings/.env
```
Comment (ronakmutha) [inline .gitignore]: Use relative paths here

File: server/manage.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 def main():
     """Run administrative tasks."""
-    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'learndjango.settings')
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'config.settings.settings')
```
Comment (ronakmutha) [inline server/manage.py]: Why `config.settings.settings` ideally it should be `config.settings.local` / `config.settings.production`

File: .gitignore (hunk 1 above)
Comment (ankushchavan1) [reply .gitignore]: updated in the new commit

File: server/manage.py (hunk 2 above)
Comment (ankushchavan1) [reply server/manage.py]: updated in the new commit as `config.settings.local` and settings file is renamed as `local.py`

**Pull request merged**